python/python/tests/test_query.py

```
@@ -1065,20 +1067,24 @@ async def test_query_serialization_async(table_async: AsyncTable):
     )
 
     # FTS queries
-    q = (await table_async.search("foo")).limit(10).to_query_object()
-    check_set_props(
-        q,
-        limit=10,
-        full_text_query=FullTextSearchQuery(columns=[], query="foo"),
-        with_row_id=False,
+    # TODO: FTS query is not a simple string now,
```

Pylint (3.3.7)

[warning] TODO: FTS query is not a simple string now,

(W0511)